src/pages/home/components/Sidebar/Sidebar.tsx

```diff
@@ -58,11 +58,23 @@ function Sidebar(): React.ReactNode {
     setInputValue(value);
   }, HALF_SECOND);
 
+  const favoriteAccounts = useMemo((): StoreAccount[] => {
+    if (isEmpty(accounts.data)) return [];
+    return accounts.data.filter((account) => account.favorite);
+  }, [accounts.data]);
+
   const renderEmptyResult = (): ReactNode => (
     <p> Não há resultados para essa busca</p>
   );
   const renderSearchData = (): ReactNode => (
     <div className={scss.accordionContainer}>
+      <SocialAccordion
+        accounts={favoriteAccounts}
+        error={false}
+        icon={<Icon icon="star-filled" size={20} />}
+        isFavoriteAccordion
+        socialMediaId="FavouriteAccounts"
+      />
```

Comment:
Acho que tomamos um gap logico aqui.

Seu accordion de favorites tinha que estar na lista de contas, não? Pelo menos na lista de favorited?

Não sei se é legal filtrar isso sempre que atualizar, temos um dado meio dificil de lidar e tratar, acredito que tudo que for favoritado tinha que já estar em um accordion proprio ja no objeto que tem as contas